tests/python/unittest/test_runtime.py

```diff
@@ -29,6 +29,17 @@ def test_libinfo_features():
     ok_(type(features) is list)
     ok_(len(features) > 0)
 
+def test_is_enabled():
```

Comment:
Can this test run in a docker image with CUDA on the machine without GPU?

Reply:
Not sure I get what's the intention.